content-src/asrouter/templates/SimpleSnippet/_SimpleSnippet.scss

```diff
@@ -88,4 +91,43 @@
   .icon {
     align-self: flex-start;
   }
+
+  &.has-section-header .innerWrapper {
+    // account for section header being 100% width
+    flex-wrap: wrap;
+  }
+
+  // wrapper div added if section-header is displayed that allows icon/text/button
+  // to squish instead of wrapping. this is effectively replicating layout behavior
+  // when section-header is *not* present.
+  .innerContentWrapper {
+    align-items: center;
+    display: flex;
+  }
+
+  .section-header {
+      flex: 0 0 100%;
+      margin-bottom: 12px;
+    }
+
+  .section-title {
```

Comment:
Vertical alignment of icon/text is slightly off
![screen shot 2019-01-22 at 10 44 49 am](https://user-images.githubusercontent.com/1455535/51546784-e32bb000-1e32-11e9-8c3c-464bda5dcc51.png)